### PR TITLE
fix: 500 error caused by duplicate email or username

### PR DESCRIPTION
**Pull request by fit2bot**
fix: 500 error caused by duplicate email or username

### CODE REVIEW

**Comment by w940853815** [comment]
image

**Review by w940853815 — APPROVED** [review]

**Review by ibuler — APPROVED** [review]

**Comment by ibuler** [comment]
Title and msg could be absolute same

**Comment by w940853815** [comment]
image

File: apps/i18n/core/zh_Hant/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1138,7 +1138,11 @@ msgstr "週期清理改密記錄和推送記錄"
 
 #: accounts/tasks/automation.py:87
 msgid "Clean change secret and push record period description"
-msgstr "系統會定期清理不必要的變更密文記錄和推播記錄，包括與變更任務、執行記錄、資產、帳戶相關的記錄。當任何這些關聯項被刪除時，相應的變更秘密和推送記錄將變得無效。因此，為了保持資料庫整潔、高效，系統預設每180天自動清理一次這些無效記錄。這種定期清理過程有助於釋放儲存空間並提高資料管理的安全性和整體效能。"
+msgstr ""
+"系統會定期清理不必要的變更密文記錄和推播記錄，包括與變更任務、執行記錄、資"
+"產、帳戶相關的記錄。當任何這些關聯項被刪除時，相應的變更秘密和推送記錄將變得"
+"無效。因此，為了保持資料庫整潔、高效，系統預設每180天自動清理一次這些無效記"
+"錄。這種定期清理過程有助於釋放儲存空間並提高資料管理的安全性和整體效能。"
```
Comment (BaiJiangJie) [inline apps/i18n/core/zh_Hant/LC_MESSAGES/django.po:1145]: 这个是不是多了其他的文案描述？

**Review by BaiJiangJie — CHANGES_REQUESTED** [review]
1

File: apps/i18n/core/zh_Hant/LC_MESSAGES/django.po (hunk 1 above)
Comment (w940853815) [reply apps/i18n/core/zh_Hant/LC_MESSAGES/django.po:1145]: 没有 只是换行

**Review by BaiJiangJie — APPROVED** [review]

**Review by BaiJiangJie — APPROVED** [review]

**Pull request merged**